tests/app/service/test_rest.py

```diff
@@ -990,6 +1033,44 @@ def test_update_service_permissions_will_add_service_permissions(client, sample_
     assert resp.status_code == 200
     assert set(result['data']['permissions']) == set([SMS_TYPE, EMAIL_TYPE, LETTER_TYPE])
 
+@pytest.mark.parametrize(
+    'email_from, should_error',
+    (
+            ('sams.sarnies', False),
```

Comment:
Worth adding a big 🥪 here to be explicit that rejecting uppercase is an expected behaviour?
```suggestion
            ('sams.sarnies', False),
            ('SAMS.SARNIES', True),            
```

Reply:
Good idea :)